src/cert.rs

```diff
@@ -12,7 +12,7 @@
 // ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
 // OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-use pki_types::{CertificateDer, DnsName};
+use pki_types::{CertificateDer, DnsName, SubjectPublicKeyInfo};
```

Comment:
I think you need to separate the `SubjectPublicKeyInfo` import into its own line, and then cfg-gate it on `alloc`. The type isn't used when the `alloc` feature isn't enabled and so generates a warning in `--no-default-features` builds.

Side-note: Once https://github.com/rustls/pki-types/pull/48 lands we'll want to switch this to use the `*Der` name.

Reply:
Gating this import in [ef13aaa](https://github.com/rustls/webpki/pull/253/commits/ef13aaa72d8377055b4b56406ab63ac774e2b358):
```diff
- use pki_types::{CertificateDer, DnsName, SubjectPublicKeyInfo};
+ #[cfg(feature = "alloc")]
+ use pki_types::SubjectPublicKeyInfoDer;
+ use pki_types::{CertificateDer, DnsName};
```

> Side-note: Once https://github.com/rustls/pki-types/pull/48 lands we'll want to switch this to use the *Der name.

Done!